platformio.ini

```diff
@@ -14,7 +15,7 @@ lib_deps =
 	lincomatic/LiquidTWI2@^1.2.7
 	olikraus/U8g2@^2.28.8
 
-[env]
+[common_embedded]
```

Comment:
actually [env] is the default build environment in pio. this is also why we inherited other environments from it (e.g. [env:mega2560]. Now other envs still inherit from the default one but we have no definition for it anymore. I would rather expect following inheritance: 
```
env
   |- embedded
      |- mega2560
      |- ...
   |- native
```

Reply:
Yeah I figured that almost all of the parameters under `common_embedded` were specific to embedded builds. I guess I could move some of the build flags to a common `env`, since we'd want the native tests to be built the same way